k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -1809,21 +1810,27 @@ private <T extends Message> void moveDangerousFolderIfNeededAndUpdateStatus(
             List<LocalMessage> messagesToNotify,
             StorageEditor storageEditor,
             LocalFolder localFolder
-    ) throws MessagingException {
```

Comment:
so we don't bubble up the exception anymore? We basically ignore it and log? Is that the desired behavior?

Reply:
Well this is my opinion. When an error occurs here, the reasons could only be two:
- Error moving a Dangerous message to Suspicious folder
- Error notifying a message
When an exception is thrown here, it will cause the app to store the original, encrypted message without any processing.
Which is not right in this case, because the fact that we arrive to this point in the execution means that we already have stored the message in the database successfully.
Then I decided to catch and not buble the exceptions because it would only cause issues.
This said, some improvement might be done in case a Dangerous message fails to be moved to Suspicious folder, like for example add a pending command to the account to do it. But this would need some investigation.

Reply:
ok clear, let's go for this approach as the worst case scenario seems to be we just  fail to move a message to the Suspicious folder and it's in any case not a big issue